### ISSUE TITLE
Add `docker run` instructions

### ISSUE THREAD

**Issue by MichaelCurrin** [linaro-its/jekyll-link-checker#9]
Hi. I'm looking for Docker run instructions for using the image - this would be useful on Docker Hub, README and/or Wiki.

**Comment by MichaelCurrin** [comment]
I found that there are instructions in the README and Wiki for running the shell script alone.

And DockerHub covers how to pull the image. https://hub.docker.com/r/linaroits/linkcheck

```sh
docker pull linaroits/linkcheck
```

But instructions are missing on how to run a container.

I suspect like this. 

```sh
docker run --rm linaroits/linkcheck
docker run --rm linaroits/linkcheck --nointernal --skip-dns-check _data/fqdn_exceptions.txt
```

That seems like the way to do it. Since the Dockerfile entrypoint goes to shell script

[Dockerfile#L81](https://github.com/linaro-its/jekyll-link-checker/blob/0b54fcf3813d69baffb1e24ac267c9dfca10720c/Dockerfile#L81)

But you'd also need some `--volume` setting in there if you want the container to read dirs like `_site` and `_data` at `.` directory.

Thanks.

**Comment by MichaelCurrin** [comment]
Also I noticed `../check-links.sh` appears in the docs and Wiki, when `check-links.sh` looks more accurate.

**Comment by pcolmer** [comment]
Hi @MichaelCurrin 

The shell script has been written to make it as easy as possible to use the Docker container. The script is included in each of Linaro's website repositories.

The expected methodology for running the script is that you are in the directory of the built site when the script is run, which is why the script is referenced as `../check-link.sh` since the script would typically be in the directory above the directory of the built site.

**Comment by MichaelCurrin** [comment]
Ok thanks.

I am not using of those repos with that script.

So it would still would help me (and others) to use for this container for one of our own sites if one extra command was added to README and Docker Hub.

Something like this.

```sh
cd my-repo
docker run --volume $PWD:/jekyll-site --rm linaroits/linkcheck -d /jekyll-site
```
```
Checking 2061 web links ...
```

As that has to be figured out and was not obvious. 

Maybe also point to `_site` and maybe add network settings? My container was getting all DNS failures and one fatal check.

Also note I found that `--volume` is needed. Otherwise I get zero links checked.

```
$ cd my-repo
$ docker run --rm linaroits/linkcheck
Container built by GitHub. Build reference: 26290ee
Linaro Link Checker (2021-02-11)
No web links to check.
```

---

I imagine that `../check-link.sh` was specific to your workflow, but if you want people outside of those sites to be able to use this tool it would help to add a note saying something like:

For a Linaro site, use a _relative_ path `../check-link.sh`.

For other sites, use an _absolute_ path to the script in the clone repo. e.g.

```sh
cd my-repos
git clone git@github.com:linaro-its/jekyll-link-checker.git
cd my-site
~/repos/jekyll-link-checker/check-link.sh
```
